gap/matrix.gi

```diff
@@ -239,8 +239,9 @@ function(ri, G)
       else   # exactly two blocks:
           # FIXME: why don't we just compute a precise set of generators of the kernel?
           # That should be easily and efficiently possible at this point, no?
+          # The kernel is abelian, so we don't need to do normal closures.
+          findgensNmeth(ri).func := FindKernelRandom;
           findgensNmeth(ri).args[1] := 7;
-          findgensNmeth(ri).args[2] := 5;
```

Comment:
I'm also not sure, what arguments of which function are stored here. Maybe some other reviewer has already worked with this?

Reply:
That should be explained in the docs of findgensNmeth. I'll check.

Reply:
These are the arguments to the function stored in `findgensNmeth(ri).method`, in this case `FindKernelRandom`.